pkg/local-storage/member/controller/volume_convert_task_worker.go

```diff
@@ -185,15 +185,14 @@ func (m *manager) volumeConvertStart(convert *apisv1alpha1.LocalVolumeConvert) e
 			for _, vol := range volList.Items {
 				if vol.Spec.VolumeGroup == lvg.Name {
 					if vol.Spec.ReplicaNumber == convert.Spec.ReplicaNumber {
-						convert.Status.State = apisv1alpha1.OperationStateInProgress
-						m.apiClient.Status().Update(ctx, convert)
+						continue
```

Comment:
`continue` is necessary, but the status should also be updated to `InProgress`

Reply:
The status will be updated to InProgress at line 203, after all lv.spec.replicaNumber was updating.